src/index.tsx

```diff
@@ -77,8 +80,8 @@ export default definePlugin((serverApi: ServerAPI) => {
 		}
 	})
 	mountManager.addMount(patchAppPage(serverApi, storage))
-	mountManager.addMount(patchHomePage(serverApi, storage))
-	mountManager.addMount(patchLibraryPage(serverApi, storage))
+	//mountManager.addMount(patchHomePage(serverApi, storage))
```

Comment:
Please do not leave commented code. VCS remember everything, so there is no need to have it.

Reply:
Sure, this was going to be a part of cleanup. I was testing multiple things so it was useful to have things to go back to.